ui/components/configuratorComponents/CodeEditor.js

```diff
@@ -19,10 +20,18 @@ const useStyles = makeStyles(theme => ({
       minWidth : "calc(50% - 175px)",
       maxWidth : "calc(50% - 175px)",
     },
+  },
+  icon : {
+    position : "absolute",
```

Comment:
Ideally, we avoid absolute positioning in general. Can this be made relative? Can we center this instead of anchoring to the bottom?

Reply:
Yes, absolute poositioning is very much needed for the context here.

Reply:
Gotcha. Ok.